dev-utils/src/commands/test_vector_command.rs

```diff
@@ -360,12 +365,52 @@ impl VectorFactory {
     }
 
     pub fn create_test_vectors(&mut self) -> Result<()> {
-        self.test_vectors.wallet_view_key = Some(hex::encode(self.wallet.view_key.serialize()));
+        self.test_vectors.wallet_view_key = Some(hex::encode(self.wallet.view_key.0.serialize()));
         let tx_id = self.create_withdraw_unbonded_tx().unwrap();
         self.create_transfer_tx(tx_id.clone())?;
         self.create_deposit_stake_tx(tx_id.clone())?;
         self.create_nodejoin_tx()?;
         self.create_unbonded_stake_tx()?;
+        println!(
+            "view secret key: {}",
+            hex::encode(self.wallet.view_key.1.serialize())
+        );
+        if let Some((ref address, ref public, ref secret)) = self.wallet.staking_address {
+            println!("staking address: {:?}", address);
+            println!("secret: {}", hex::encode(secret.serialize()));
+            println!("public key: {}", hex::encode(public.serialize()));
+        }
+
+        for (address, public, secret) in self.wallet.transfer_addresses.iter() {
+            println!("transfer address");
+            println!("mainnet: {}", address.to_cro(Network::Mainnet).unwrap());
+            println!(
+                "public testnet: {}",
+                address.to_cro(Network::Testnet).unwrap()
+            );
+            let xonly =
+                XOnlyPublicKey::from_secret_key(&Secp256k1::new(), &SecretKey::from(secret));
+            println!("secret: {}", hex::encode(secret.serialize()));
+            println!("public key: {}", hex::encode(public.serialize()));
+            println!("X only public key: {}", hex::encode(&xonly.serialize()));
+        }
+
+        /*
+                let xonly = XOnlyPublicKey::from_secret_key(&Secp256k1::new(), &SecretKey::from(&priv_key));
```

Comment:
is this comments necessary?

Reply:
oh, forgot to save the last revision -- a good catch, thanks!